### PR TITLE
timesheet's dummy data

### PR DESCRIPTION
**Pull request by bbcatalan**

### CODE REVIEW

File: app/Models/Employee.php [hunk 1]
```diff
@@ -8,8 +8,7 @@
 class Employee extends Model
 {
     use HasFactory;
-
-    public static function newEmployee($data)
+        public static function newEmployee($data)
```
Comment (djsaballa) [inline app/Models/Employee.php]: remove the space

File: app/Models/Timesheet.php [hunk 2]
```diff
@@ -8,4 +8,13 @@
 class Timesheet extends Model
 {
     use HasFactory;
+    public static function newTimesheet($data)
```
Comment (djsaballa) [inline app/Models/Timesheet.php:12]: enter here after use HasFactory;

File: database/migrations/2021_11_10_061749_create_timesheets_table.php [hunk 3]
```diff
@@ -14,13 +14,13 @@ class CreateTimesheetsTable extends Migration
...
-            $table->string('time_out');
-            $table->string('break_start');
-            $table->string('break_end');
-            $table->timestamps();
+            $table->date('date')->useCurrent();
+            $table->time('time_in')->useCurrent();
+            $table->time('time_out')->useCurrent();
+            $table->time('lunch_start')->useCurrent();
+            $table->time('lunch_end')->useCurrent();
+            $table->nullableTimestamps();
```
Comment (djsaballa) [inline database/migrations/2021_11_10_061749_create_timesheets_table.php]: turn into timestamps(); only

Comment (bbcatalan) [reply database/migrations/2021_11_10_061749_create_timesheets_table.php]: eradicate the "nullable"?

File: employee.csv [hunk 4]
```diff
@@ -1,4 +1,4 @@
-lastname,firstname,email,password
+Lastname,Firstname,Email,Password
```
Comment (djsaballa) [inline employee.csv]: Bakit mo ginawang capital? Kung ano yung nakalagay sa database natin, dapat ganern din ilalagay dito para magkaparehas lang

**Pull request merged**